mypy/test/teststubgen.py

```diff
@@ -443,7 +444,9 @@ def h():
 
         assert_equal(remove_misplaced_type_comments(original), dest)
 
-    def test_common_dir_prefix(self) -> None:
+    @pytest.mark.skipif(sys.platform == 'win32',
```

Comment:
We use `@unittest.skipIf` elsewhere and it would be better to use it here for consistency.